packages/fuels-signers/src/provider.rs

```diff
@@ -36,6 +36,12 @@ impl From<WalletError> for ProviderError {
         ProviderError::WalletError(e.to_string())
     }
 }
+
+impl From<ProviderError> for Error {
+    fn from(e: ProviderError) -> Self {
+        Error::ProviderError(e.to_string())
+    }
+}
```

Comment:
Shouldn't this be in `error.rs` with all the other error conversion functions?

Reply:
Since `From<WalletError> for Error` is implemented in `wallet.rs` i thought this would be appropriate. 
It kind of makes sense for me this way, since `Error` is a global error type so it shouldn't be aware of local error types, rather the local error types should know how to convert to it.

Reply:
You're right! Makes me think we should implement the other `From<LocalErrorTypes>` inside their respective files.